.github/workflows/pytest-all.yml

```
@@ -45,4 +45,4 @@ jobs:
         run: python setup.py develop --user
 
       - name: Run tests with pytest
-        run: pytest -v -s --cov=. --nbval --ignore=docs --ignore-glob="pynucastro/library/tabular/*.ipynb" --ignore=examples/rxn-network-integration.ipynb --ignore=examples/o16o16_rates.ipynb
+        run: pytest -v -s --cov=. --nbval --ignore=docs --ignore-glob="pynucastro/library/tabular/*.ipynb" --ignore=examples/rxn-network-integration.ipynb --ignore=examples/o16o16_rates.ipynb --ignore=pynucastro/networks/tests/der_net.py --ignore=examples/*.py
```

This is what I've been using locally:
```suggestion
        run: pytest -v -s --cov=pynucastro --nbval --ignore=docs --ignore-glob="pynucastro/library/tabular/*.ipynb" --ignore=examples/rxn-network-integration.ipynb --ignore=examples/o16o16_rates.ipynb
```

---

okay, that works.  But I think that we need to explicitly ignore `der_net.py` still

---

For coverage, or so pytest doesn't try to run it? We ignore everything matching `*/tests/*` in .coveragerc, and pytest only collects tests in files matching `test_*.py` or `*_test.py`.